lib/ascii_binder/engine.rb

```diff
@@ -189,12 +189,12 @@ def page(args)
       args[:breadcrumb_root], args[:breadcrumb_group], args[:breadcrumb_subgroup], args[:breadcrumb_topic] = extract_breadcrumbs(args)
 
       args[:breadcrumb_subgroup_block] = ''
-      args[:subtopic_shim]             = ''
       if args[:breadcrumb_subgroup]
         args[:breadcrumb_subgroup_block] = "<li class=\"hidden-xs active\">#{args[:breadcrumb_subgroup]}</li>"
-        args[:subtopic_shim]             = '../'
       end
 
+      args[:subtopic_shim] = '../' * (args[:topic_id].split('::').length - 2)
```

Comment:
You still need to catch the case where there are no subtopic dirs. You might want to follow this with a line like:

`args[:subtopic_shim] = '' if args[:subtopic_shim].nil?`